NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
                 "timestamp": pumpDateStr,
                 "bolusiob": status.iob,
             ],
-            "reservoir": status.reservoirRemainingUnits,
+            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
```

Comment:
Interesting, well let's change the data type to Double in the message then, which should help. 

Reservoir values are only given to the 0.1, though, which shouldn't have precision issues with a float32...

> On Jun 24, 2016, at 4:04 PM, Pete Schwamb <notifications@github.com> wrote:
> 
> In NightscoutUploadKit/NightscoutUploader.swift:
> 
> > @@ -169,7 +169,7 @@ public class NightscoutUploader: NSObject {
> >                  "timestamp": pumpDateStr,
> >                  "bolusiob": status.iob,
> >              ],
> > -            "reservoir": status.reservoirRemainingUnits,
> > +            "reservoir": Double(round(status.reservoirRemainingUnits*1000)/1000,
> Well, it manifests like this: 0.07500000000000001 instead of 0.075. Not a serious issue in terms of accuracy.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>